tests/ValidationBusinessLogic.Tests/LineItemDetailsInvalidPasses.cs

```diff
@@ -0,0 +1,155 @@
+﻿using InvoiceGenerator.BusinessLogic;
+using InvoiceGenerator.ViewModels;
+using Xunit;
+
+namespace ValidationBusinessLogic.Tests
+{
+    public class LineItemDetailsInvalidPasses
+    {
+        [Theory]
+        [InlineData("", "", "")]
+        [InlineData(null, null, null)]
+        public void Check_If_Error_Is_Returned_When_Invalid_Details_Input(string lineItemDescription, string lineItemCost, string lineItemQuantity)
+        {
+            // Arrange
+            var expectedResult = new LineItemViewModelValidationResult()
+            {
+                LineItemDescriptionValidationMessage = "Line Item Description must not be empty!",
+                LineItemCostValidationMessage = "Line Item Cost must not be empty!",
+                LineItemQuantityValidationMessage = "Line Item Quantity must not be empty!"
+            };
+            var input = new LineItemViewModel()
+            {
+                Description = lineItemDescription,
+                CostPer = lineItemCost,
+                Quantity = lineItemQuantity
+            };
+            var SUT = new ValidationBLogic();
+            // Act
+            var result = SUT.validateLineItemDetails(input);
+            // Assert
+            Assert.Equal(expectedResult.LineItemDescriptionValidationMessage, result.LineItemDescriptionValidationMessage);
+            Assert.Equal(expectedResult.LineItemCostValidationMessage, result.LineItemCostValidationMessage);
+            Assert.Equal(expectedResult.LineItemQuantityValidationMessage, result.LineItemQuantityValidationMessage);
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void Check_LineItemDescription_Null_Or_Empty(string lineItemDescription)
+        {
+            // Arrange
+            var expectedResult = new LineItemViewModelValidationResult()
+            {
+                LineItemDescriptionValidationMessage = "Line Item Description must not be empty!",
+                LineItemCostValidationMessage = "No error!",
```

Comment:
:pick: these `"No Error!"` strings shouldn't be required as they are being set by the (implicit) default constructor. If you remove this assignment (i.e. `LineItemCostValidationMessage = "No error!"` at line 45), does the test still pass?